week_2/task_6_sequence_reverse/task_6.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
```

Comment:
@Ypostolov  using namespace is bad practice.
Please try to keep Google coding style across your tasks